proxy/src/telemetry/metrics/mod.rs

```diff
@@ -456,7 +456,7 @@ impl Aggregate {
                 let labels = Arc::new(ResponseLabels::fail(res));
                 self.update(|metrics| {
                     metrics.response_total(&labels).incr();
-                    metrics.response_latency(&labels).add(latency::Ms(fail.since_request_open));
+                    metrics.response_latency(&labels).add(fail.since_request_open.into());
```

Comment:
to avoid having to type `into` a whole lot, you can actually just move this to `add` by changing the type to `add<U: Into<V>>(u: U)` and calling `into()` there instead...